merkledag/ipld.md

```diff
@@ -302,11 +302,50 @@ On the subject of integers, there exist a variety of formats which represent int
 
 IPLD supports a variety of serialized data formats through [multicodec](https://github.com/jbenet/multicodec). These can be used however is idiomatic to the format, for example in `CBOR`, we can use `CBOR` type tags to represent the merkle-link, and avoid writing out the full string key `mlink`. Users are encouraged to use the formats to their fullest, and to store and transmit IPLD data in whatever format makes the most sense. The only requirement **is that there MUST be a well-defined one-to-one mapping with the IPLD Canonical format.** This is so that data can be transformed from one format to another, and back, without changing its meaning nor its cryptographic hashes.
 
+## Serialised CBOR with tags
+
+IPLD objects can be represented using cbor using the tags described below when possible. Tags are defined in [RFC 7049 section 2.4](http://tools.ietf.org/html/rfc7049#section-2.4):
+
+- `<tag-base58>`: the byte string that follows (major type 2) is to be interpreted as a text string instead (major type 3). This text string is the base58 encoded version of the byte string (using the IPFS alphabet).
+
+- `<tag-link-object>`:  an array (major type 4) of two or three elements (link prefix, link hash (optional) and map) must follow:
+
+    - The link prefix must be an integer representing the first path of the link, or a text string appended at the beginning of the link. Available integers are:
+        - `1`: represents the prefix `/ipfs/`
```

Comment:
yeah a multiaddr-like thing would make this much simpler, eh? (and not require two CBOR tags)